.github/workflows/checks.yml

```diff
@@ -106,6 +106,47 @@ jobs:
         with:
           command: check
 
+  uniswap:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - uses: dcarbone/install-jq-action@v2
+      - name: Install cargo-risczero
+        uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-risczero@0.19
+      - uses: actions/cache@v3
+        with:
+          key: ${{ runner.os }}-${{ github.job }}-${{ github.head_ref }}
+          path: ./target
+      - name: Build citrea
+        run: SKIP_GUEST_BUILD=1 make build
+      - name: Install node dependencies
+        working-directory: ./bin/citrea/tests/evm/uniswap
+        run: npm install
+      - name: Run uniswap tests
+        run: |
+          RUST_LOG=off ./target/debug/citrea --da-layer mock --rollup-config-path bin/citrea/configs/mock/sequencer_rollup_config.toml --sequencer-config-path bin/citrea/configs/mock/sequencer_config.toml --genesis-paths bin/test-data/genesis/demo-tests/mock &
+          sleep 2
+          RUST_LOG=off ./target/debug/citrea --rollup-config-path bin/citrea/configs/mock/rollup_config.toml --genesis-paths bin/test-data/genesis/demo-tests/mock &
+          ./bin/citrea/publish_block.sh &
```

Comment:
after this sleep 2 and run publish_da_block as well

Reply:
Done